tests/rfc_7.test.ts

```diff
@@ -69,17 +64,27 @@ describe("with commit labels", () => {
   describe("with matching GitHub labels", () => {
     it("adds GitHub labels that match commit labels", () => {
       dm.danger.github.api.issues.getLabels.mockImplementationOnce(() => ({
-        data: [
-          { name: "wontfix" },
-          { name: "Messaging" },
-          { name: "Auctions" }
-        ]
+        data: [{ name: "wontfix" }, { name: "Messaging" }, { name: "Auctions" }],
       }))
       return rfc7().then(() => {
         expect(dm.danger.github.api.issues.addLabels).toHaveBeenCalledWith({
-          owner: 'artsy', repo: 'eigen', number: 1234, labels: ['Auctions']
+          owner: "artsy",
+          repo: "eigen",
+          number: 1234,
+          labels: ["Auctions"],
         })
       })
     })
+
+    it("doesn't add existing GitHub labels on the issue", async () => {
+      dm.danger.github.issue.labels = [{ name: "Auctions" }]
+      dm.danger.github.api.issues.getLabels.mockImplementationOnce(() => ({
+        data: [{ name: "wontfix" }, { name: "Messaging" }, { name: "Auctions" }],
+      }))
+
+      return rfc7().then(() => {
+        expect(dm.danger.github.api.issues.addLabels).not.toHaveBeenCalled()
+      })
+    })
   })
```

Comment:
This is the only new change, the repo doesn't have prettier on it, so that needs to happen at some point.